apps/meteor/app/statistics/server/lib/SAUMonitor.ts

```diff
@@ -143,7 +143,8 @@ export class SAUMonitorClass {
 				projection: { loginToken: 1 },
 			});
 			if (!session?.loginToken) {
-				throw new Error('Session not found');
+				logger.error('Session not found during logout', { userId, sessionId });
```

Comment:
now that you solved the issue would be the case of keeping the throw?

thanks to `throw` we finally solved the issue